ckanext/publications_qld_theme/templates/package/group_list.html

```diff
@@ -1,31 +1,30 @@
 {% extends "package/read_base.html" %}
 {% import 'macros/form.html' as form %}
 
+{% set default_group_type = h.default_group_type('group') %}
+
 {% block primary_content_inner %}
-  <h2 class="hide-heading">{{ _('Categories') }}</h2>
+  <h2 class="hide-heading">{{ h.humanize_entity_type('group', default_group_type, 'page title') or _('Categories') }}</h2>
 
-  {% if h.check_access('package_update', {'id':c.pkg_dict.id }) %}
-    {% set groups = h.groups_available() %}
-    {% if groups %}
-      <form class="add-to-group" method="post">
-        <div class="form-group">
-          <select id="field-add_group" class="form-control" name="group_added">
-            {% for option in groups %}
-              <option value="{{ option.id }}"> {{ option.display_name }}</option>
-            {% endfor %}
-          </select>
-        </div>
-        <button type="submit" class="btn btn-primary" title="{{ _('Associate this category with this dataset') }}">{{ _('Add to category') }}</button>
-      </form>
-    {% endif %}
+  {% if group_dropdown %}
```

Comment:
Where is this from? Can you add comment on 'what' we altered from upstream.

Reply:
Synced with core 2.11 template. We replace "group" with "category" for Publications.

Reply:
Added a comment

Reply:
It seems like we could instead achieve this by overriding the `humanize_entity_type` helper.

Reply:
The helper is not currently sufficient. I have raised https://github.com/ckan/ckan/issues/8682 to address this.